client-v2/src/shared/components/Collection.js

```diff
@@ -58,17 +56,16 @@ const collectionDetail = ({ collection, stage }: Props) =>
     <span>Waiting for collection</span>
   );
 
-const createMapStateToProps = () => {
```

Comment:
How come the change from a factory here?

Reply:
selectById doesn't require memoization, it's a lookup :)

Reply:
Simple as that!